native_client/swift/deepspeech_ios/DeepSpeech.swift

```diff
@@ -139,7 +139,7 @@ public struct DeepSpeechTokenMetadata {
 */
 public struct DeepSpeechCandidateTranscript {
     /// Array of DeepSpeechTokenMetadata objects
-    private(set) var tokens: [DeepSpeechTokenMetadata] = []
+    public private(set) var tokens: [DeepSpeechTokenMetadata] = []
```

Comment:
"public private", is it on purpose?

Reply:
It means the getter is public but the setter is private.